test/functional/ui/syntax_conceal_spec.lua

```diff
@@ -0,0 +1,402 @@
+local helpers = require('test.functional.helpers')
+local Screen = require('test.functional.ui.screen')
+local clear, feed, execute = helpers.clear, helpers.feed, helpers.execute
+local insert = helpers.insert
+
+describe('Screen', function()
+  local screen  
+
+  before_each(function() 
+    clear()
+    screen = Screen.new()
+    screen:attach()
+    screen:set_default_attr_ignore( {{bold=true, foreground=255}} )
+    screen:set_default_attr_ids( {{foreground = Screen.colors.LightGrey, background = Screen.colors.DarkGray}} )
+  end)
+  
+  after_each(function()
+    screen:detach()
+  end)  
+
+  describe("match and conceal", function()
+
+    before_each(function()
+      execute("let &conceallevel=1")
+    end)
+
+    describe("multiple", function()
```

Comment:
if you want, you can change this to "multiple double ampersands" (or prehaps "multiple double characters", the fact they are ampersands are not that interesting, as one would expect this to work with any characters) and change the first it to "and staying below all concealed groups".